cloud/pkg/devicecontroller/apis/devices/v1alpha1/device_instance_types.go

```diff
@@ -136,6 +136,15 @@ type Twin struct {
 	Reported TwinProperty `json:"reported,omitempty"`
 }
 
+type DeviceState struct {
+	Reported DeviceOnOffline `json:"state,omitempty"`
+}
+
+type DeviceOnOffline struct {
+	LightState string `json:"light_state, omitempty"`
```

Comment:
what do we mean by LightState ? all devices don't have such kind of state .

Reply:
When I run led example successfully, I got some data in edge database. According to  the device table's data in edge database, I wrote this structure, maybe we should reconsider this table's fields.